README.md

```diff
@@ -535,6 +535,17 @@ This branch can then be rebased and a pull request can be opened from it as usua
 Note: when using mappings in your `vendor_package`, the patches will be relative to the mapped directory.
 Hence, for upstreaming, you might need to use `git am --directory=<mapping.from>` instead of plain `git am`.
 
+### `completion` --- Generate shell completion script
+
+The `bender completion <SHELL>` command prints a completion script for the given shell.
+
+Supported shells:
+- `bash`
+- `elvish`
+- `fish`
+- `powershell`
+- `zsh`
+
```

Comment:
Can you add how to enable the completion for the different shells?

I think simply mentioning the script needs to be placed into a file and then sourced in the shell, preferably directly in the rc file, would suffice. This should avoid confusion by new users.